requirements.txt

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4~=4.6
 cld2_cffi~=0.1
-spacy~=2.0
+spacy~=2.0.17
```

Comment:
maybe remove the tilde if we need an exact version of the library ?

Reply:
It determines "at least this version", so, not a specific version. It leaves room to move up (if bugs get fixed).